### PR TITLE
Split testCMCEMGDrivenArm into two to speed it up

### PR DESCRIPTION
**Pull request by klshrinidhi**
Similar to #1127.

### CODE REVIEW

File: Applications/CMC/test/testCMCEMGDrivenArm_Millard.cpp [hunk 1]
```diff
@@ -30,41 +30,15 @@
...
-
-    cout << "Done" << endl;
-
-    return 0;
-}
-
 void testEMGDrivenArm() {
     cout<<"\n******************************************************************" << endl;
-    cout << "*                         testEMGDrivenArm                       *" << endl;
+    cout << "*                 testEMGDrivenArm_Millard                       *" << endl;
```
Comment (tkuchida) [inline Applications/CMC/test/testCMCEMGDrivenArm_Millard.cpp]: testEMG… -> testCMCEMG…?

File: Applications/CMC/test/testCMCEMGDrivenArm_Millard.cpp [hunk 2]
```diff
@@ -80,3 +54,26 @@ void testEMGDrivenArm() {
     const string& muscleType = cmc.getModel().getMuscles()[0].getConcreteClassName();
     cout << "\ntestEMGDrivenArm "+muscleType+ " passed\n" << endl;
```
Comment (tkuchida) [inline Applications/CMC/test/testCMCEMGDrivenArm_Millard.cpp]: testEMG… -> testCMCEMG…?

**Comment by tkuchida** [comment]
`</review>`  :+1:

File: Applications/CMC/test/testCMCEMGDrivenArm_Millard.cpp (hunk 1 above)
Comment (klshrinidhi) [inline Applications/CMC/test/testCMCEMGDrivenArm_Millard.cpp]: Done.

File: Applications/CMC/test/testCMCEMGDrivenArm_Millard.cpp (hunk 2 above)
Comment (klshrinidhi) [inline Applications/CMC/test/testCMCEMGDrivenArm_Millard.cpp]: Done.

**Pull request merged**